src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__college_assessment_dashboard.sql

```diff
@@ -202,3 +205,87 @@ left join
     and e.expected_scope = p.scope
     and e.expected_subject_area = p.subject_area
 where e.expected_test_type = 'Practice'
+union all
+select
+    e.academic_year,
+    e.region,
+    e.schoolid,
+    e.school_abbreviation,
+    e.student_number,
+    e.lastfirst,
+    e.grade_level,
+    e.enroll_status,
+    e.cohort,
+    e.entrydate,
+    e.exitdate,
+    e.sped,
+    e.c_504_status,
+    e.lep_status,
+    e.advisor_lastfirst,
+    e.contact_id,
+    e.ktc_cohort,
+    e.courses_course_name,
+    e.teacher_lastfirst,
+    e.sections_external_expression,
+    'Official' as expected_test_type,
+    'AP' as expected_scope,
+    case
+        e.courses_course_name
+        when 'AP English Language and Composition'
+        then 'English Language and Composition'
+        when 'AP World History: Modern'
+        then 'World History: Modern'
+        when 'AP US History'
+        then 'US History'
+        when 'AP Psychology'
+        then 'Psychology'
+        when 'AP African American Studies'
+        then 'African American Studies'
+        when 'AP Calculus AB'
+        then 'Calculus AB'
+        when 'AP Chemistry'
+        then 'Chemistry'
+        when 'AP Computer Science Principles'
+        then 'Computer Science Principles'
+        when 'AP Physics'
+        then ''

```

Comment:
Q: Why are these empty quotes?

Reply:
Added this to our O3 next week. I couldnt decide which one to use from the list. Like, US History has 3 options.